.github/workflows/breaking-change-check.yml

```diff
@@ -24,14 +24,14 @@ jobs:
       matrix: ${{ steps.get-matrix.outputs.matrix }}
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE so job can access it
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           fetch-depth: 0
       # Ensure node version is great enough
       - name: Use Node.js
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
-          node-version: '20.x'
+          node-version: 'lts/*'
```

Comment:
I'm hesitant about using LTS on anything CI where we should be as stable as possible and try pinning dependencies that may shift under us. We will hit those issues where suddenly half the PRs start failing or snapshots are partially updating if the linux version bump has some big changes.

Is the purpose to improve performance (prevent the linux machines from having to take time installing the right version)?

Reply:
I understand we want to control the deps as much as possible.
For the case of CI though, we should be using LTS node regardless. This change keeps it in sync.

The purpose for this change is so we don't need as much maintenance time. This will always use LTS.

Reply:
As an aside, we could also make this a matrix and run the whole thing against multiple node versions...

Reply:
> For the case of CI though, we should be using LTS node regardless

Why is this? I feel we _shouldn't_ be using LTS for CI. At all times we should have a stable development platform that devs know will work and can contribute changes that remains unblocked. Separate from that, we then upgrade our development contract when new versions of node are LTS _only after_ we've verified new versions work.

I would definitely be in favor of action though that is kicked off nightly/automatically when we new versions of node are released and verifies CI works.

Reply:
The crux here is that LTS will migrate the node version as LTS roles automatically, which means there's no way it will get forgotten. It means one less thing to update.

If we're keeping on top of dependencies internally, then there will never be an issue. If, however, there does happen to be a break when a new version of node comes out and we have not been ready for it, we have a means to ensure it is caught.